tests/concepts/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_executable(test_concepts_View test_View.cpp)
+target_link_libraries(test_concepts_View PRIVATE Kokkos::utils GTest::gtest_main Kokkos::kokkoscore)
+add_test(NAME test_concepts_View COMMAND $<TARGET_FILE:test_concepts_View>)
```

Comment:
Make basic macro to add test. Argument (namespace, class that we test, ...). Link, add test, add to doc.